tests/unit/test_command_containers.py

```diff
@@ -24,7 +29,7 @@ def test_render(runner, tag):
 def test_render_with_env_var_override(runner):
     with tempfile.NamedTemporaryFile() as tmpfile:
         with mock.patch.dict(os.environ, {'GIT_BRANCH': 'other'}):
-            result = runner.invoke(cli, ['render', f'--dockerfile={tmpfile.name}'])
+            result = runner.invoke(cli, ['render', f'--dockerfile={tmpfile.name}'], catch_exceptions=False)
             assert result.exit_code == 0
```

Comment:
This one should assert that the GIT_BRANCH is substituted in to the rendered Dockerfile, I don't know why it doesn't.